recipes/eko/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "eko" %}
+{% set version = "0.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/N3PDF/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f7f6d7faed207bbbdbbbf003348e1afdcf823196ecea0156730f0097a3686ef3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win or py<38]
+
+requirements:
+  host:
+    - python
+    - poetry
+    - pip
+  run:
+    - python
+    - scipy
+    - lz4
+    - numba
+    - pyyaml
+    - numpy
+
+test:
+  imports:
+    - eko
+  source_files:
+    - tests
+    - pyproject.toml
+  requires:
+    - pytest < 7
+    - pytest-cov
+    - pytest-env
+  commands:
+    - pytest tests -k "not test_pegasus_path"
```

```suggestion
    - pytest tests -k "not test_pegasus_path"
    - pip check
    - ekonav --help
```
To check if 1. the deps are correct 2. if the CLI is working.